### ISSUE TITLE
Fail to run snyk with Node 12.1.0

### ISSUE THREAD

**Issue by blootsvoets** [snyk/snyk#555]
- `node -v`: v12.1.0
- `npm -v`: 6.9.0
- `snyk -v`: 1.171.1
- Command run: 
`snyk`
`snyk -h`

### Expected behaviour
Usage information about the CLI

### Actual behaviour
The command prints
```
module.apply is not a function
```

### Steps to reproduce
Install with `npm install -g snyk`.
```
snyk -h --debug
  snyk help { _: [ 'help' ], help: 'help', debug: true } +0ms
TypeError: module.apply is not a function
    at Object.method (/Users/***/.config/yarn/global/node_modules/snyk/src/lib/hotload.js:15:21)
    at /Users/***/.config/yarn/global/node_modules/snyk/src/cli/index.ts:27:29
    at Generator.next (<anonymous>)
    at /Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:4:12)
    at runCommand (/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:34:12)
    at /Users/***/.config/yarn/global/node_modules/snyk/src/cli/index.ts:144:17
    at Generator.next (<anonymous>)
    at /Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:8:71
  snyk analytics {
  args: [ 'help' ],
  command: 'bad-command',
  metadata: {
    'error-message': 'module.apply is not a function',
    error: 'TypeError: module.apply is not a function\n    at Object.method ' +
      '(/Users/***/.config/yarn/global/node_modules/snyk/src/lib/hotload.js:15:21)\n' +
      '    at ' +
      '/Users/***/.config/yarn/global/node_modules/snyk/src/cli/index.ts:27:29\n' +
      '    at Generator.next (<anonymous>)\n    at ' +
      '/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:8:71\n' +
      '    at new Promise (<anonymous>)\n    at __awaiter ' +
      '(/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:4:12)\n' +
      '    at runCommand ' +
      '(/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:34:12)\n' +
      '    at ' +
      '/Users/***/.config/yarn/global/node_modules/snyk/src/cli/index.ts:144:17\n' +
      '    at Generator.next (<anonymous>)\n    at ' +
      '/Users/***/.config/yarn/global/node_modules/snyk/dist/cli/index.js:8:71',
    'error-code': undefined,
    command: 'help'
  },
  version: '1.171.1',
  os: 'macOS Mojave',
  nodeVersion: 'v12.1.0',
  id: '***',
  ci: false,
  durationMs: 45
} +0ms
  snyk sending request to: https://snyk.io/api/v1/analytics/cli +0ms
  snyk request body size: 1047 +0ms
  snyk gzipped request body size: 415 +0ms
  snyk not using proxy +0ms

  snyk Exit code: 2 +0ms
```

**Comment by lili2311** [comment]
👋 We have updated our test matrix to test against node 12, all the tests pass. Please could you confirm if you are still experiencing this issue? We have released many fixes and updates recently.

**Comment by blootsvoets** [comment]
The issue is resolved, tested with version 1.184.0. Thanks for the update!
